### ISSUE TITLE
incorrect image tag after crictl pull

### ISSUE THREAD

**Issue by gouyang** [kubernetes-incubator/cri-tools#229]
After pull the image `redis:alpine`, it's actually `redis:latest` shows in `crictl images`.

```
# crictl pull docker.io/library/redis:alpine
Image is update to date for docker.io/library/redis@sha256:0d9e5a4a3072948e1460a510bb07b2636d5c160ee84edb98daac8e779756f08d
# crictl images
IMAGE                     TAG                 IMAGE ID            SIZE
docker.io/library/redis   latest              861cc310cd916       111MB

# crictl -v
crictl version 1.0.0-alpha.0
# crio -v
crio version 1.9.0-dev
commit: "33e105710288002370be5b0f2eec4ddfe2834227"

```

**Comment by feiskyer** [comment]
@gouyang Thanks of reporting the problem. This is indeed a bug in crictl.

**Comment by feiskyer** [comment]
Filed #230 to fix the problem.

**Comment by feiskyer** [comment]
Fixed by #231
